### ISSUE TITLE
How to improve the accuracy of reconstruction

### ISSUE THREAD

**Issue by 627633854** [torrvision/spaint#10]
When I scanned a complex model, I could see the details of the model in the spaint window, but the accuracy of the model displayed after the reconstruction was much worse. 
Is there any way to improve it?

**Comment by sgolodetz** [comment]
Change the voxel size down to something sensible in spaintgui's main.cpp possibly - do you have a screenshot?

**Comment by 627633854** [comment]
yes, i scan a Hulk model, but lost a lot of details.
![image](https://user-images.githubusercontent.com/25810393/41074633-93d9a6de-6a3b-11e8-90bd-393d33236540.png)  
and the result 
![image](https://user-images.githubusercontent.com/25810393/41074656-b5870ab0-6a3b-11e8-87ca-3029f9946b91.png)

**Comment by sgolodetz** [comment]
What's the scale of the model in comparison to the voxel size?

**Comment by 627633854** [comment]
The size of model is 25cm,21cm,13cm，is it small?

**Comment by sgolodetz** [comment]
Yes, that's pretty small - if you're using 0.5cm voxels, that means your model is only 50 voxels across. Plus, there's depth accuracy to consider - at that scale, the fine details in the model are unlikely to be captured well by a standard depth camera, so it's not just a matter of making the voxel size smaller (although you could try that, to get some improvement).
